contracts/LiquidityBridgeContract.sol

```diff
@@ -276,13 +331,23 @@ contract LiquidityBridgeContract {
             increaseBalance(quote.liquidityProviderRskAddress, refundAmount);
             int256 remainingAmount = transferredAmount - int(refundAmount);
             
+			// SDL: Should we refund ANY amount. Maybe some wallets do round 
+			// the number to X decimal places and the user cannot send exactly
+			// the amount he wants. If the extra amount sent is "dust", then
+			// maybe we should not attempt to refund.
```

Comment:
you can set it this way: you take the minimum gas price, and you multiply it by 4500 (50% of the cost of a value transfer call).
This means that overhead of the payment will never be more than 50%.

But it's not a good idea to get the minimum gas price every time from the BlockHeader contract, because getting the minimum has cost of ~4000 gas.  I can't use the current transaction gasPrice, because that is controlled by the caller.
The easier way is just compute DUST once according to the current gasPrice, and leave like that.
Another possibility is to use Bitcoin dust value. (defined in Bitcoin source code)
One really cool but overengineered possibility is to update the DUST variable (According to minimumGasPrice) pseudo-randomly checking some bytes from the previous block hash (using the BLOCKHASH opcode, which costs only 20 gas). 
For example if (prevBlockhash % 100 == 0) recomputeDust()